physical_properties/pKa/microstates/SM01_microstates.csv

```diff
@@ -1,11 +1,9 @@
 microstate ID,canonical isomeric SMILES
 SM01_micro001,c1cc2c(cc1O)c3c(o2)C(=[OH+])NCCC3
 SM01_micro002,c1cc2c(cc1[O-])c3c(o2)C(=NCCC3)[O-]
-SM01_micro003,c1cc2=[O+]C3=C(NCCCC3=c2cc1O)[O-]
```

Comment:
I'm going to be honest, I'm not good enough at reading SMILES strings to check these so I'm going to trust your visualization and the openeye code we worked on together.

Reply:
OK. I am also relying on participants to tell us, if they catch another mistake.